eng/common/scripts/Verify-Links.ps1

```diff
@@ -248,7 +268,11 @@ function GetLinks([System.Uri]$pageUri)
 
   return $links;
 }
-
+$urls = @()
+$suffixArray = $suffixToScan -split "," | ForEach-Object { $_.Trim() }
```

Comment:
This completely breaks the scenario where someone wants to pass an actual url as input which we do in various places. We should make the caller compute the set of files or urls they want to check.

Reply:
We can take both Urls and (path and suffix) pair. We kind of serve as batch work script. There are limited usage of urls besides of local testing. Most cases I can think of are "give me the folder, and scan all the broken links please". Have URLs and filter the right thing before using the scripts is kind of twisting to me.

Reply:
There are other places like https://github.com/Azure/azure-sdk-tools/blob/master/eng/pipelines/githubio-linkcheck.yml#L14 which use the url and we should definitely keep that as an option. I think we should have the list of url's be handled by the caller instead of backing in the traversal logic in the script as that removes a lot of flexibility.